src/og/control/timeline/TimelineControl.js

```diff
@@ -26,7 +26,7 @@ class TimelineControl extends Control {
         }
         super(options);
 
-        let currentDate = options.currentDate || new Date();
+        let currentDate = options.current || new Date();
```

Comment:
I think currentDate more convenient name for the option. If we change the name it's better to change all currentDate in the code.